frontend/src/patientApp/timeOfTest/TestResult.tsx

```diff
@@ -11,7 +11,7 @@ const TestResult = () => {
   const patient = useSelector<RootState, Patient>((state) => state.patient);
   const fullName = formatFullName(patient as any);
   const dateTested = new Date(patient.lastTest.dateTested).toLocaleDateString();
-  const deviceType = patient.lastTest.deviceTypeModel;
+  const deviceType = patient.lastTest.deviceTypeName;
```

Comment:
Are there any database instances that include the model but not the name?

Reply:
Not in this view, we are only showing the device name. However in the PDF we show both device name and test name.

Reply:
I decided to keep the deviceTypeModel in the api response for backward compatibility with older UIs